ansible/roles/ci/update_dependencies/files/load_repositories.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
please put a comment next to it as we do on other files

Reply:
Done